runtime/altair/src/migrations.rs

```diff
@@ -58,676 +58,427 @@ pub type UpgradeAltair1031 = (
 /// Altair.
 #[cfg(feature = "testnet-runtime")]
 pub type UpgradeAltair1031 = (
-	// TODO: Verify that these ones can be removed from Algol upgrade, since the first 2 fail
-	// runtime_common::migrations::nuke::Migration<crate::Loans, RocksDbWeight, 1>,
-	// runtime_common::migrations::nuke::Migration<crate::InterestAccrual, RocksDbWeight, 0>,
```

Comment:
Note: No nuke required on `Algol` for `Loans` and `InterestAccrual` as these are already at the latest `StorageVersion`